pkg/synopsysctl/constants.go

```diff
@@ -53,34 +62,34 @@ var baseChartRepository = "https://sig-repo.synopsys.com/sig-cloudnative"
 const AlertPostSuffix = "-alert"
 
 // Alert Helm Chart Constants
-var alertVersion = "5.3.2"
+var alertVersion = alert.DefaultFlagTree.Version
```

Comment:
Can we do an Opposite i.e. having all the versions defined here instead of having in it's own folder? This will make us to modify the default versions in one place instead of multiple files. And I'm not sure whether it will create a cyclic dependency?

Reply:
I can try but I think that might cause an import cyle problem:
* synosysctl imports pkg/alert
* pkg/alert imports synospysctl

A different solution could be to put all the helm chart defaults in a separate package

Reply:
```
import cycle not allowed
package github.com/blackducksoftware/synopsysctl/cmd/synopsysctl
	imports github.com/blackducksoftware/synopsysctl/pkg/synopsysctl
	imports github.com/blackducksoftware/synopsysctl/pkg/alert
	imports github.com/blackducksoftware/synopsysctl/pkg/synopsysctl
```